UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1685,6 +1697,14 @@ private void AdminSmash()
 		{
 			AdminCommandsManager.Instance.CmdHealMob(gameObject);
 		}
+
+		private void ClownAbuseScoreEvent(DamageType damageType, GameObject abuser)
+		{
+			if(damageType == DamageType.Clone || damageType == DamageType.Oxy || damageType == DamageType.Radiation) return;
+			if(abuser.TryGetComponent<PlayerScript>(out var script) == false) return;
```

Comment:
Abuser can be `null`, so there should be a check for that.

Reply:
done.